### ISSUE TITLE
Undefined offset / SWL_CustomTexts.php on line 62

### ISSUE THREAD

**Issue by mwjames** [SemanticMediaWiki/SemanticWatchlist#59]
```
( ! ) Notice: Undefined offset: 2 in /home/mw_nichts/htdocs/w/extensions/SemanticWatchlist/includes/SWL_CustomTexts.php on line 62
Call Stack
```

Filed by @DestructionOfPlanetJupiter from [0] in #57.

The offending `$this->customTexts[$parts[0]][$parts[1]] = $parts[2];` can be found in [1].

[0] http://pastebin.com/es3Gr9A7
[1] 
https://github.com/SemanticMediaWiki/SemanticWatchlist/blob/master/includes/SWL_CustomTexts.php#L62

**Comment by mwjames** [comment]
related to #23

**Comment by DestructionOfPlanetJupiter** [comment]
SWL_CustomTexts.php:
(line 52)
```
 $row = $dbr->selectRow(
                        'swl_groups',
                        'group_custom_texts',
                        array( 'group_name' => $this->group->getName() ),
                        'SWL::initCustomTexts'
                );
```
Those rows above are missing. The outcome if obvious:
$row -> null
$set -> null (naturally)
and so on and so forth from this point onward.

**Comment by Fannon** [comment]
I can confirm this issue, this is happening to me when running the SMW rebuildDatabase maintainence script

**Comment by tosfos** [comment]
Fixed in 3d538e4113a63e54a3992dedc3506bf2fb003d88
